docs/changelog/v-0-11-0.rst

```diff
@@ -9,5 +9,20 @@ Frame
 -----
 
 - |fix| A list of frames now displays properly in a Jupyter lab (#2222).
+
+
+
+General
+-------
+
+- |api| Properties ``dt.stype.min`` and ``dt.stype.max`` are now equal to
+  ``False`` and ``True`` respectively, instead of integers 0 and 1 (#2231).
```

Comment:
Should we add "For boolean columns..."?

Reply:
You right, it's a typo

Reply:
*You're

what's up with the typos today? 😉